README.md

```diff
@@ -24,6 +24,11 @@ The Godot Language Server should be running separately. Here are the steps to se
 4. Set "Exec Path" to the path of your Zed executable
 5. Set "Exec Flags" to `{project} {file}:{line}:{col}`
 
+If you have installed Godot and Zed via Flatpak on Linux, use the following parameters:
+
+4. Set "Exec Path" to `flatpak-spawn`
+5. Set "Exec Flags" to `--host flatpak run dev.zed.Zed {project} {file}:{line}:{col}`
+
```

Comment:
```suggestion
If you have installed Godot and Zed via Flatpak on Linux, use the following parameters:

1. Set "Exec Path" to `flatpak-spawn`
2. Set "Exec Flags" to `--host flatpak run dev.zed.Zed {project} {file}:{line}:{col}`

```

Reply:
Steps 4 and 5 meant that steps 1-3 were needed and only 4 and 5 had to be replaced :laughing:
But of course; I'm totally fine with that :+1: